ui-kit/src/main/kotlin/org/p2p/uikit/utils/text/TextViewCellModel.kt

```diff
@@ -143,5 +178,12 @@ private data class InitialTextStyle(
         backgroundTint = textView.backgroundTintList,
         gravity = textView.gravity,
         hintTextColors = textView.hintTextColors,
+        textViewAutoSizeConfiguration = TextViewAutoSizeConfiguration(
+            autoSizeTextType = textView.autoSizeTextType,
+            autoSizeMinTextSize = textView.autoSizeMinTextSize,
+            autoSizeMaxTextSize = textView.autoSizeMaxTextSize,
+            autoSizeStepGranularity = textView.autoSizeStepGranularity,
+            typedValue = TypedValue.COMPLEX_UNIT_PX,
+        ),
     )
 }
```

Comment:
I think this class is getting overloaded. 
Maybe, it's better to create a folder for `TextViewCellModel ` and clean here a little bit?

Reply:
I'll clean it next pr